.all-contributorsrc

```diff
@@ -0,0 +1,42 @@
+{
+  "projectName": "acadivity",
+  "projectOwner": "Ajitesh13",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "Ajitesh13",
+      "name": "Ajitesh Panda",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/42615579?v=4",
+      "profile": "https://bugzilla.mozilla.org/user_profile?user_id=650533",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "routrohan",
+      "name": "Rohan Rout",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/32121879?v=4",
+      "profile": "https://github.com/routrohan",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "agg-shambhavi",
+      "name": "Shambhavi Aggarwal",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/48705124?v=4",
+      "profile": "https://www.linkedin.com/in/shambhavi-aggarwal-437804179/",
+      "contributions": [
+        "code"
```

Comment:
Please add documentation here.

Reply:
Oh, sorry I forgot! ok, Merge this one now, I am creating another issue to fix this

Reply:
Okay.